db/agents.json

```diff
@@ -638,6 +876,16 @@
         "Player FM"
       ]
     },
+    {
+      "regex": "^Plex.+iOS",
+      "name": "Plex",
+      "type": "Mobile App",
+      "os": "iOS",
```

Comment:
if we need more OS examples, I can hit Plex pretty easily from an Apple TV, a browser, or a Roku.